### PR TITLE
feat(nms): Redesign popover and account settings

### PR DESCRIPTION
**Pull request by thmsschmitt**
## Summary
Implements design changes from https://www.figma.com/file/Ec5jkliFb4i4OYGdI9AkYk/Combining-Multiple-Portals
Followup of #12402

## New Design
Screenshot 2022-04-08 at 13 10 50
Screenshot 2022-04-08 at 13 10 37

The account settings are not pixel perfect because I tried to reuse existing elements to keep the UI consistent.
Screenshot 2022-04-08 at 13 10 13

## Test Plan
Go to the account settings and make sure that changing the password still works.

### CODE REVIEW

**Comment by andreilee** [comment]
Will review after #12402 lands.

File: nms/packages/magmalte/app/components/NetworkSelector.js [hunk 1]
```diff
@@ -26,9 +26,9 @@ import React from 'react';
 import SettingsEthernetIcon from '@material-ui/icons/SettingsEthernet';
 import Text from '../../fbc_js_core/ui/components/design-system/Text';
 import Tooltip from '@material-ui/core/Tooltip';
-import classNames from 'classnames';
 import useMagmaAPI from '../../api/useMagmaAPI';
 
+import symphony from '../../fbc_js_core/ui/theme/symphony';
```
Comment (andreilee) [inline nms/packages/magmalte/app/components/NetworkSelector.js]: You'll need to restructure `fbc_js_core/ui/theme` a bit.

For context, 'symphony' was an old project that was discontinued, and now partially lives on as a zombie within Magma (or fbc-js-core). We don't really want any references to a 'symphony' to continue living on.

File: nms/packages/magmalte/fbc_js_core/ui/components/ProfileButton.js [hunk 2]
```diff
@@ -106,10 +95,14 @@ const ProfileButton = () => {
     location.pathname.includes(adminUrl) ||
     location.pathname.includes(settingsUrl);
 
+  const hasAccountSettings = !ssoEnabled;
```
Comment (andreilee) [inline nms/packages/magmalte/fbc_js_core/ui/components/ProfileButton.js]: This looks like something that could potentially be re-used. Could we move this into `AppContext`?

**Review by andreilee — COMMENTED** [review]
I have some early comments. Overall this looks great

File: nms/packages/magmalte/app/components/NetworkSelector.js (hunk 1 above)
Comment (thmsschmitt) [reply nms/packages/magmalte/app/components/NetworkSelector.js]: I'll use `app/theme/default` instead and I'll work on a separate PR to get rid of `fbc_js_core/ui/theme`.

**Comment by thmsschmitt** [comment]
> Will review after #12402 lands.

@andreilee #12402 has been merged

**Review by andreilee — APPROVED** [review]

**Pull request merged**